tests/openapi/test_block_hashes.py

```
@@ -0,0 +1,124 @@
+    drop_collection(collection_name)
+    response = request_with_validation(
+        api="/collections/{collection_name}", method="PUT",
+        path_params={"collection_name": collection_name},
+        body={"vectors": {"size": 2, "distance": "Dot"}, "shard_number": 3},
+    )
+    assert response.ok
+    yield
+    drop_collection(collection_name)
+
+
+def hashes(collection_name, count=16, filter=None):
```

Ruff (0.16.3)

[error] Function argument `filter` is shadowing a Python builtin

(A002)